src/Org.Quickstart.API/Models/CouchbaseConfig.cs

```diff
@@ -8,7 +8,11 @@ public class CouchbaseConfig
 		public string ScopeName { get; set; }
 	    public string RestEndpoint { get; set; }
 
-        public string ConnectionString { get; set; }	
+		public bool IgnoreRemoteCertificateNameMismatch { get; set; }
+		public bool HttpIgnoreRemoteCertificateMismatch { get; set; }
```

Comment:
What if I don't want to ignore that?

Reply:
What would be the correct way of accepting the certificate?

Reply:
set it to false

Reply:
You would have to do something similar to Java if you wanted to check all handshake requests against a custom certificate.  The code isn't great, can be confusing, and cause performance issues because you have to compare every handshake against a file on disk and secure the file on disk.  It's not fun to do custom SSL Certificate Stores.